frontend/src/metabase/dashboard/dashboard.js

```diff
@@ -1165,9 +1165,13 @@ const loadingDashCards = handleActions(
 
 const loadMetadataForDashboard = dashCards => (dispatch, getState) => {
   const metadata = getMetadata(getState());
+
+  // the flatmap here can cause nulls to be added to the list of cards, filter them.
   const queries = dashCards
     .flatMap(dc => [dc.card].concat(dc.series))
+    .filter(dc => dc != null)
```

Comment:
@paulrosenzweig - I added this line because the flatMap was adding nulls to the queries collection, which would blow up the `Question` constructor below. Can you run this branch / give it a look over and see if the behavior seems right to you?

Reply:
It looks like there were two separate causes for the nulls. This worked great, but I tweaked the code slightly and committed onto this branch.

This is also a good reminder that we should have e2e coverage of auto dashboards. I'll make an issue for that.

Reply:
Awesome, thank you. I just smacked it with a very blunt hammer. :grin: